dll/win32/shell32/lang/el-GR.rc

```diff
@@ -396,7 +396,7 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
     ICON IDI_SHELL_OPEN_WITH, -1, 8, 12, 21, 20
     LTEXT "Επιλέξτε το πρόγραμμα με το οποίο θέλετε να ανοίξετε αυτό το αρχείο:", -1, 44, 12, 211, 10
-    LTEXT "Αρχείο:    %s", 14001, 44, 25, 188, 20
+    LTEXT "Αρχείο:    ", 14001, 44, 25, 188, 20
```

Comment:
![image](https://github.com/reactos/reactos/assets/2107452/f42fd41e-3a8e-4ead-82aa-37c18f440192)

Reply:
It's addendum to 4ff1aa5fac0 (r30784) where the initial dialog update were not applied to el-GR and 4af7888b29d (0.4.14-dev-1492-g4af7888) where the update was partially applied (AUTOCHECKBOX).